src/MessageParser.php

```diff
@@ -95,16 +95,26 @@ class MessageParser {
     public function __invoke($message)
     {
         $content = strtolower($message->content);
+
         // Grab the first 4 chars for our trigger
         $trigger = substr($content, 0, 5);
+
         // Grab everything after the first 5 chars
         $query = substr($content, 5);
 
+        // Check if version is available in command
+        $pattern = '/\b([6-9]\.x)\b/'; // only match 6.x to 9.x
+        preg_match($pattern, $query, $matches);
+        if ($version = $matches[0] ?? "9.x") // default to 9.x if no version is given
```

Comment:
+1 for checking valid versions, if they give an invalid version we should return the URL without a version (Defaulting to latest)